lib/srv/app/aws/endpoints.go

```diff
@@ -182,29 +181,32 @@ var signingNameToEndpointsID = map[string]string{
 	"aws-marketplace":                       marketplacecatalog.EndpointsID,
 	"awsiottwinmaker":                       iottwinmaker.EndpointsID,
 	"awsmigrationhubstrategyrecommendation": migrationhubstrategyrecommendations.EndpointsID,
-	"awsmobilehubservice":                   mobile.EndpointsID,
-	"awsproton20200720":                     proton.EndpointsID,
-	"awsssooidc":                            ssooidc.EndpointsID,
-	"awsssoportal":                          sso.EndpointsID,
-	"detective":                             detective.EndpointsID,
-	"ecr":                                   ecr.EndpointsID,
-	"ecr-public":                            ecrpublic.EndpointsID,
-	"elastic-inference":                     elasticinference.EndpointsID,
-	"iot-jobs-data":                         iotjobsdataplane.EndpointsID,
-	"iot1click":                             iot1clickdevicesservice.EndpointsID,
-	"iotdata":                               iotdataplane.EndpointsID,
-	"iotdeviceadvisor":                      iotdeviceadvisor.EndpointsID,
-	"ioteventsdata":                         ioteventsdata.EndpointsID,
-	"iotfleethub":                           iotfleethub.EndpointsID,
-	"iotsecuredtunneling":                   iotsecuretunneling.EndpointsID,
-	"iotwireless":                           iotwireless.EndpointsID,
-	"lex":                                   lexmodelsv2.EndpointsID,
-	"mediatailor":                           mediatailor.EndpointsID,
-	"memorydb":                              memorydb.EndpointsID,
-	"mobiletargeting":                       pinpoint.EndpointsID,
-	"pricing":                               pricing.EndpointsID,
-	"sagemaker":                             sagemaker.EndpointsID,
-	"ses":                                   ses.EndpointsID,
-	"sms-voice":                             pinpointsmsvoice.EndpointsID,
-	"timestream":                            timestreamquery.EndpointsID,
+	// AWS mobile service deprecated since v1.55.0.
+	// Constant copied from mobile.EndpointsID:
+	// https://github.com/aws/aws-sdk-go/blob/019bed03fa64f3edad98bba262d41d58eb2b9fee/service/mobile/service.go#L33-L34
+	"awsmobilehubservice": "mobile",
```

Comment:
Do we really need to keep this?

Reply:
I'll leave that to @greedy52 - hard for me to say.

Reply:
If AWS deprecated it, I would assume it no longer works

Reply:
Thanks for tracking this! How did you find out?

Seems a few more services will get deprecated:
https://cloudperceptor.medium.com/aws-has-deprecated-several-products-without-prior-notice-473e16457e46

> If AWS deprecated it, I would assume it no longer works

Seems most existing resources will still work:
https://dev.to/sirlawdin/upcoming-aws-service-deprecations-5ak1

I guess existing users will be forced to use older SDKs.

> Do we really need to keep this?

It's good to have it. However, these mappings are only used in "legacy" endpoint URL mode, which is rarely used when the new "HTTPS_PROXY" does not work. So I am ok with losing them for easier maintenance.

Reply:
Thanks for clarifying it.

So, if your app requires an older SDK due to a deprecated service and there's an active CVE impacting it, you'll need to handle your own SDK distribution. Thanks, AWS ❤️

Reply:
Thanks for the replies and review, STeve.

service/mobile is sorted _forever_ as far as I'm concerned, but others will likely break in future updates. If you have time, feel free to remove (or hardcode) whatever services you see fit.

> Thanks for tracking this! How did you find out?

Easy - it breaks the build because the service/mobile package doesn't exist anymore. I also inspect a good chunk of the release notes before upgrading.